cli/cli/kurtosis_config/overrides_objects/v4/kubernetes_cluster_config_v4.go

```diff
@@ -0,0 +1,17 @@
+package v4
+
+/*
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! WARNING !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+                           DO NOT CHANGE THIS FILE!
+  If you change this file, it will break config for users who have instantiated an
+           overrides file with this version of config overrides!
+    Instead, to make changes, you will need to add a new version of the config
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! WARNING !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+*/
+
+type KubernetesClusterConfigV4 struct {
+	KubernetesClusterName  *string `yaml:"kubernetes-cluster-name,omitempty"`
+	StorageClass           *string `yaml:"storage-class,omitempty"`
+	EnclaveSizeInMegabytes *uint   `yaml:"enclave-size-in-megabytes,omitempty"`
+	EngineNodeName         *string `yaml:"engine-node-name,omitempty"`
```

Comment:
engine node name is added via v4 config to have engine scheduled on same node for a consistent log view (commented in code) - will need to document that if this changes, users might see inconsistent logs